package/bin/chat/user_command_handler.js

```diff
@@ -335,6 +335,22 @@
           return this.displayMessage('notice', this.listInstalledScripts());
         }
       });
+      this._addCommand('show-script', {
+	description: "show source code for script",
+	params: ['scriptName'],
+	usage: '<script name>',
+	category: 'scripts',
+	run: function() {
+          const script = this.chat.scriptHandler.getScriptByName(this.scriptName);
```

Comment:
There's a lot of inconsistent indentation, would you mind cleaning this up a little? Generally this follows the Google javascript style of a 2-space indent for new blocks.